exams/pearson/writers_test.py

```diff
@@ -186,7 +187,7 @@ def test_write_skips_invalid_rows(self):  # pylint: disable=no-self-use
 
 
 @ddt.ddt
-class CDDWriterTest(TSVWriterTestCase):
+class CDDWriterTest(TSVWriterTestCase, TestCase):
```

Comment:
theoretically after my PR about the tests optimizations in case of problems you should see the tests fail, so you are fine here, but you are right, we should document what I just said.
Feel free to leave as is if you want.